### ISSUE TITLE
Update for 3.2.1-2

### ISSUE THREAD

**Issue by madileweb** [Vitaliy-1/JATSParserPlugin#51]
Hi, I'm testing the plugin on 3.2.1-2 and get multiple errors.

Is planned launch an update for this version of OJS?

Thank you in advance

**Comment by Vitaliy-1** [comment]
Hi @madileweb,

What release did you test? The list of all available ones and correspondent compatibility with OJS version: https://github.com/Vitaliy-1/JATSParserPlugin/releases

**Comment by madileweb** [comment]
Hi!, the last one 2.1.9-2 for 3.2.1.

**Comment by Vitaliy-1** [comment]
It's fully compatible. 
What are the multiple errors that you see? Can you post the error(s) from PHP log here?

**Comment by madileweb** [comment]
Hi @ Vitaliy-1,

Sorry, it was my fault. We have OJS 3.1.2-1 in production and were testing an older version without full text and now We want to migrate to OJS 3.2.1-2 and I was trying to test the new version with Galley and got many errors.
So I ran the new plugin version just fine, but I have some doubts.

We use Manuscript theme for our journal and what I could see is full text shows ok on Health Science theme. What change I must to do for show full text on other theme? and PDF is no more generate on fly too?

Thank you!!

**Comment by Vitaliy-1** [comment]
Yes, PDF is also generated statically. There is a correspondent checkbox in Publication Settings.

Full-text should be shown as with other themes, like Health Sciences, but may require additional styling.
